docs-ref-services/latest/app-configuration-readme.md

```diff
@@ -1,19 +1,17 @@
 ---
-title: 
+title: Use App Configuration client library for JavaScript
```

Comment:
It is supposed to change through our script. 
@ramya-rao-a Do we have a plan to release the app-configuration? Seems like the content here is out of date. It should update through our release pipeline.

Reply:
We don't have a planned update for App Config. I would recommend we keep the changes in this PR for App Config